### PR TITLE
Updating Dockerfile to allow for vaxrank PDF report generation:

### PR DESCRIPTION
**Pull request by julia326**
- adding non-Python dependencies
- adding a font config file needed for good kerning in output PDF (Ubuntu issue)

Tested by creating a local Docker image with this Dockerfile and running vaxrank.

### CODE REVIEW

File: tools/docker/run/Dockerfile [hunk 1]
```diff
@@ -19,11 +24,14 @@ RUN sudo bash -c "DEBIAN_FRONTEND=noninteractive apt-get install --yes --allow-u
...
   adduser --uid 20042 --disabled-password --gecos '' biokepi && \
   passwd -l biokepi && \
   chown -R biokepi:biokepi /home/biokepi"
 USER biokepi
 ENV HOME /home/biokepi
 WORKDIR /home/biokepi
 RUN opam init -a --yes --comp 4.02.3
+
+COPY fonts.conf $HOME/.config/fontconfig/fonts.conf
+RUN ln -s $HOME/.config/fontconfig/fonts.conf $HOME/.fonts.conf
```
Comment (smondet) [inline tools/docker/run/Dockerfile]: (Just out of curiosity)  is there a know way to make this setup system-wide?

That would make it a bit less fragile (?).

Comment (julia326) [inline tools/docker/run/Dockerfile]: System-wide meaning outside of Linux? Annoyingly, not really - HTML/PDF rendering is proving to cause platform-specific problems, so this solution is only necessary on Ubuntu. But it shouldn't break other things. What other platforms do we run biokepi on?

Comment (smondet) [inline tools/docker/run/Dockerfile]: No I meant for the whole system as opposed to for the current user (`$HOME` is `/home/biokepi`).

**Comment by smondet** [comment]
The Travis error seems to be (again) because of the new version of `omake`.

File: tools/docker/run/Dockerfile [hunk 2]
```diff
@@ -19,11 +24,14 @@ RUN sudo bash -c "DEBIAN_FRONTEND=noninteractive apt-get install --yes --allow-u
...
   adduser --uid 20042 --disabled-password --gecos '' biokepi && \
   passwd -l biokepi && \
   chown -R biokepi:biokepi /home/biokepi"
 USER biokepi
 ENV HOME /home/biokepi
 WORKDIR /home/biokepi
 RUN opam init -a --yes --comp 4.02.3
+
+# Copy local fonts config over, also needed for PDF gen
+COPY fonts.conf /etc/fonts/local.conf
```
Comment (smondet) [inline tools/docker/run/Dockerfile]: Cool :+1: good to know :)

**Comment by julia326** [comment]
Partially fixes https://github.com/hammerlab/epidisco/issues/93

**Pull request merged**